torch/csrc/distributed/rpc/tensorpipe_agent.h

```diff
@@ -2,6 +2,7 @@
 
 #ifdef USE_TENSORPIPE
 
+#include <gtest/gtest_prod.h>
```

Comment:
cc @lw to check if this is OK.

Reply:
First time I see this, I assume it's for the `FRIEND_TEST` macro. In that case it looks fine to me. My only concern is whether we're sure that gtest is always available at build time? (Maybe there's a "no-test" build mode where gtest could be missing?)